### DISCUSSION TITLE
Please fork K-9 Mail 🙏

### DISCUSSION THREAD

**Discussion by josephg5** [FossifyOrg/General-Discussion#251] [General]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/General-Discussion/issues) or [closed](https://github.com/FossifyOrg/General-Discussion/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/General-Discussion/discussions) or [closed](https://github.com/FossifyOrg/General-Discussion/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs), in the [README](https://github.com/FossifyOrg/General-Discussion/blob/master/README.md) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report may get dismissed otherwise.**
- [X] This issue contains only one feature request.
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Which apps should have this feature?

All Fossify Apps

### Feature description

Mozilla takeover has ruined K-9 Mail. I think we should fork the old K-9 before Mozilla destroyed it. So many complaints, and they are making it worse. It's not the same app anymore, but a completely different one.

### Why do you want this feature?

I think since you took over simple apps, and made them better, while keeping the simple philosophy, perhaps you might consider adding an email app to your applist. Thanks.

**Comment by naveensingh** [comment]
Thanks but there are no plans for a mail app. See https://github.com/orgs/FossifyOrg/discussions/68
